### DISCUSSION TITLE
xDrip+ to Nightscout uploader: not setting eventType on bolus field?

### DISCUSSION THREAD

**Discussion by butako** [NightscoutFoundation/xDrip#3648] [Bug reports]
Hi, 
I am unable to search for Bolus treatments on the Nightscout API, specifically this call returns nothing: 
https://secret.com//api/v1/treatments.json?find[created_at][$gte]=2024-08-01&find[created_at][$lte]=2024-08-30&find[eventType]=Bolus&count=131072&token=secret

However if I use this API, it does return something:
https://secret.com//api/v1/treatments.json?find[insulin][$gte]=0&token=secret

I notice that the eventType field is "\<none\>", as shown here:

[{"_id":"86fec31fd87543c6b7cfc6a3","timestamp":1724929251289,"eventType":"\<none\>","enteredBy":"xdrip","uuid":"1234","insulin":1,"insulinInjections":"[]","created_at":"2024-08-29T11:00:51.000Z","sysTime":"2024-08-29T12:00:51.289+0100","utcOffset":0,"carbs":null}]

I don't know if this is a bug in xDrip or Nightscout, but the EventType should probably be "Bolus".

FYI, I am using https://github.com/creepymonster/nightscout-to-libreview and this fetches the Insulin treatments via the first API query I pasted above.

xDrip: latest stable
Nightscout: 14.2.6

**Comment by Navid200** [comment]
14.2.6 is not the latest stable release of Nightscout.
